spi/core-spi/src/test/java/org/eclipse/dataspaceconnector/spi/query/ReflectionBasedQueryResolverTest.java

```diff
@@ -0,0 +1,136 @@
+/*
+ *  Copyright (c) 2020 - 2022 Microsoft Corporation
+ *
+ *  This program and the accompanying materials are made available under the
+ *  terms of the Apache License, Version 2.0 which is available at
+ *  https://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  SPDX-License-Identifier: Apache-2.0
+ *
+ *  Contributors:
+ *       Microsoft Corporation - initial API and implementation
+ *
+ */
+package org.eclipse.dataspaceconnector.spi.query;
+
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Test;
+
+import java.util.Collection;
+import java.util.Comparator;
+import java.util.List;
+import java.util.Objects;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+import java.util.stream.Stream;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+class ReflectionBasedQueryResolverTest {
+
+    private QueryResolver<FakeItem> queryResolver = new ReflectionBasedQueryResolver<>(FakeItem.class);
+
+    @Test
+    void verifyQuery_noFilters() {
+        var stream = IntStream.range(0, 10).mapToObj(FakeItem::new);
```

Comment:
nit: as this data is used by all tests(except one) , may be we can move it to a helper method. But this is more of a coding style so I leave it up to you.

No need to change if you prefer this way. Thanks

Reply:
makes sense, but this way it shows explicitly what's the id of the FakeItem. It's used later e.g. in tests with sorting. It's also short so I would keep it this way